tests/test_views.py

```diff
@@ -1,6 +1,6 @@
-import json
+from anime.app import db
+from anime.schemas import AnimeSchema
 
```

Comment:
You don't need the extra empty line 3 since the anime module and .factories file are in the local project.

Reply:
The imports above sorted by isort and I usually don't look after isort. Anyways, I will reflect it :)